### ISSUE TITLE
Windows wheels have duplicate RECORD entries

### ISSUE THREAD

**Issue by SpecLad** [python-zeroconf/python-zeroconf#1125]
```console
$ wget https://files.pythonhosted.org/packages/bf/93/f88c7cba61437ab58acced48977f48ac2c51758de
...
$ bsdtar -xOf zeroconf-0.47.1-cp311-cp311-win_amd64.whl zeroconf-0.47.1.dist-info/RECORD | sort
zeroconf-0.47.1.dist-info/COPYING,sha256=nDX6DHmRuyQHZHGsKrsMpriibTk7t6AEGT0Y9x5XFKA,24838
zeroconf-0.47.1.dist-info/METADATA,sha256=dpJ1m7meSnCNhVVJSm3Uml3XLUf0O8gpGwl0DMX2cPE,6297
zeroconf-0.47.1.dist-info/RECORD,,
zeroconf-0.47.1.dist-info/WHEEL,sha256=q-KLDJQebzIZyKLb3oMnX-cCkloRZtwcIVhfy70qOs8,98
zeroconf/asyncio.py,sha256=XdNUuLYJpFtTTHJaeiIaHiB7kCD7I7On3XpuQ6AwrOo,10945
zeroconf/asyncio.py,sha256=XdNUuLYJpFtTTHJaeiIaHiB7kCD7I7On3XpuQ6AwrOo,10945
zeroconf/_cache.cp310-win_amd64.pyd,sha256=JuMRQL-gqoy9uC2UBqDz4Hj6yyMT0mshPnRWDzTc-Lk,109056
zeroconf/_cache.c,sha256=3Gvew8YN5Smg3W6PnmoJL6_Sq2nntd4dcSgWuZA6dZk,547535
zeroconf/_cache.c,sha256=3Gvew8YN5Smg3W6PnmoJL6_Sq2nntd4dcSgWuZA6dZk,547535
zeroconf/_cache.pxd,sha256=WNNHYzn_My5jlx68ENHYxz5JdytkTBOm69g3uqiG27U,539
zeroconf/_cache.pxd,sha256=WNNHYzn_My5jlx68ENHYxz5JdytkTBOm69g3uqiG27U,539
zeroconf/_cache.py,sha256=vuRS-nDcOiVpZROj1jxjQOQxycdvMKvc7G4iWwTywwg,8721
zeroconf/_cache.py,sha256=vuRS-nDcOiVpZROj1jxjQOQxycdvMKvc7G4iWwTywwg,8721
zeroconf/const.py,sha256=AGsAEulRDfVwnAPyCWwuYHZ7TbMOfn3pcDt7uQA4uEQ,4586
zeroconf/const.py,sha256=AGsAEulRDfVwnAPyCWwuYHZ7TbMOfn3pcDt7uQA4uEQ,4586
zeroconf/_core.py,sha256=PLhMAcSqDtn1KjLLX-eRrpqi9PCDqbZAiZD6_oBb7C0,39019
zeroconf/_core.py,sha256=PLhMAcSqDtn1KjLLX-eRrpqi9PCDqbZAiZD6_oBb7C0,39019
zeroconf/_dns.cp310-win_amd64.pyd,sha256=YTGbbnNtEiHF8ZBvfF8hHQVvDvmR0EkL9Ry9WXOwLuU,246272
zeroconf/_dns.c,sha256=bo8a5L1yyuLgsm_HpUVz9pDke_A_7wpNvh5_-8XsXXc,1521986
zeroconf/_dns.c,sha256=bo8a5L1yyuLgsm_HpUVz9pDke_A_7wpNvh5_-8XsXXc,1521986
zeroconf/_dns.pxd,sha256=arX-KTvGTdzzhd7mgVSQLtf3keTS_oG-_bTDOgdxwdc,2048
zeroconf/_dns.pxd,sha256=arX-KTvGTdzzhd7mgVSQLtf3keTS_oG-_bTDOgdxwdc,2048
zeroconf/_dns.py,sha256=n30uenSds6iSWdGqVofmDBjUGuZiZ3m3Tr3m1hBuYzU,18702
zeroconf/_dns.py,sha256=n30uenSds6iSWdGqVofmDBjUGuZiZ3m3Tr3m1hBuYzU,18702
zeroconf/_exceptions.py,sha256=TKx3Z8mvPsG93p50PEiM7mCH7WbWgp9GZeWb0JWvF2s,2234
zeroconf/_exceptions.py,sha256=TKx3Z8mvPsG93p50PEiM7mCH7WbWgp9GZeWb0JWvF2s,2234
zeroconf/_handlers.py,sha256=FSS74jufM6lLgy3HUpFgU1LsUfT-O9wIm0qjO5kHBfQ,26372
zeroconf/_handlers.py,sha256=FSS74jufM6lLgy3HUpFgU1LsUfT-O9wIm0qjO5kHBfQ,26372
zeroconf/_history.py,sha256=ClAiuPP1dFJtlnYjJLRvEOHYNfLAfHlecPbrrnLDWD8,2986
zeroconf/_history.py,sha256=ClAiuPP1dFJtlnYjJLRvEOHYNfLAfHlecPbrrnLDWD8,2986
zeroconf/__init__.py,sha256=KybhNqDX9KCk__1Zq6wicU67cpaNjK5P0wDBFso-7io,3992
zeroconf/__init__.py,sha256=KybhNqDX9KCk__1Zq6wicU67cpaNjK5P0wDBFso-7io,3992
zeroconf/_logger.py,sha256=r6OAmCLGWuFIHnERTxgPifKtSZ72Q9yZLfs8ugiIpHQ,3066
zeroconf/_logger.py,sha256=r6OAmCLGWuFIHnERTxgPifKtSZ72Q9yZLfs8ugiIpHQ,3066
zeroconf/_protocol/incoming.cp310-win_amd64.pyd,sha256=LUhIApnypmNQ3hG0hm7cGE4P_c_Y1MO8-XrRcgC7mHQ,113664
zeroconf/_protocol/incoming.c,sha256=gYvaWFvgppvrfkrlbT9U4IWaDZuyH4Yik0PxvDeyIcI,681199
zeroconf/_protocol/incoming.c,sha256=gYvaWFvgppvrfkrlbT9U4IWaDZuyH4Yik0PxvDeyIcI,681199
zeroconf/_protocol/incoming.pxd,sha256=qxEATWlGXOncbzwJXtPaxx4wMU4S_x6FUxJG1RRpMSU,2208
zeroconf/_protocol/incoming.pxd,sha256=qxEATWlGXOncbzwJXtPaxx4wMU4S_x6FUxJG1RRpMSU,2208
zeroconf/_protocol/incoming.py,sha256=bjCAy3q1wKnNEz8bnbImFfefPvupbKa9WFO9bgAnWEc,13783
zeroconf/_protocol/incoming.py,sha256=bjCAy3q1wKnNEz8bnbImFfefPvupbKa9WFO9bgAnWEc,13783
zeroconf/_protocol/__init__.py,sha256=Grw4SdBaV4HOx-MzaUL--AYLmxhYxNpOhNLQp7UWyB0,992
zeroconf/_protocol/__init__.py,sha256=Grw4SdBaV4HOx-MzaUL--AYLmxhYxNpOhNLQp7UWyB0,992
zeroconf/_protocol/outgoing.cp310-win_amd64.pyd,sha256=RqdGhV0mfLtdp-A6fUOjI_qMwvaycQK8J8vKqK434Q4,153600
zeroconf/_protocol/outgoing.c,sha256=c5hBrYdXTVe1Lt3SftEY_peVcfTdfzqTRskmgh4ojNo,880518
zeroconf/_protocol/outgoing.c,sha256=c5hBrYdXTVe1Lt3SftEY_peVcfTdfzqTRskmgh4ojNo,880518
zeroconf/_protocol/outgoing.pxd,sha256=ZxKbO854kw9Q-TUFxgbQgT4r1MSDnfF6JY00wGsX_Mw,2018
zeroconf/_protocol/outgoing.pxd,sha256=ZxKbO854kw9Q-TUFxgbQgT4r1MSDnfF6JY00wGsX_Mw,2018
zeroconf/_protocol/outgoing.py,sha256=yxPEFR1Fq49GGWSS2P0DycMJUbe12K_pVH0EZu4_O0g,18216
zeroconf/_protocol/outgoing.py,sha256=yxPEFR1Fq49GGWSS2P0DycMJUbe12K_pVH0EZu4_O0g,18216
zeroconf/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
zeroconf/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
zeroconf/_services/browser.py,sha256=7Z11zfz-Gdk5hEFTvr3WcYf3QG99KMWWAXWTim3hVn4,22390
zeroconf/_services/browser.py,sha256=7Z11zfz-Gdk5hEFTvr3WcYf3QG99KMWWAXWTim3hVn4,22390
zeroconf/_services/info.py,sha256=mF5ykf8Y_QYAeBBe1Tnw5zmyFX-SMZ4LdgspqC5xTP0,21852
zeroconf/_services/info.py,sha256=mF5ykf8Y_QYAeBBe1Tnw5zmyFX-SMZ4LdgspqC5xTP0,21852
zeroconf/_services/__init__.py,sha256=Lcu_14K4anCiVEokwSYiV55_GsmQYaKsADykpIdvopA,2429
zeroconf/_services/__init__.py,sha256=Lcu_14K4anCiVEokwSYiV55_GsmQYaKsADykpIdvopA,2429
zeroconf/_services/registry.py,sha256=MGkyZnTFmg50pn-S-Jsjr62KYMFIVSvdzDHfmx1Mc8g,3926
zeroconf/_services/registry.py,sha256=MGkyZnTFmg50pn-S-Jsjr62KYMFIVSvdzDHfmx1Mc8g,3926
zeroconf/_services/types.py,sha256=BMsJ9VdedAk_mrUGiUNdcuTGTs-Cv-D0V7blcJg81YM,3086
zeroconf/_services/types.py,sha256=BMsJ9VdedAk_mrUGiUNdcuTGTs-Cv-D0V7blcJg81YM,3086
zeroconf/_updates.py,sha256=ub3KAQFgoOQGHHO5H7djGn6j3tkJ49I1DVUz-uSNX_o,2989
zeroconf/_updates.py,sha256=ub3KAQFgoOQGHHO5H7djGn6j3tkJ49I1DVUz-uSNX_o,2989
zeroconf/_utils/asyncio.py,sha256=OUiVqCjmFh3NuT4_nfI_CUXSZ2rnR25PY_fu2SES75Y,4254
zeroconf/_utils/asyncio.py,sha256=OUiVqCjmFh3NuT4_nfI_CUXSZ2rnR25PY_fu2SES75Y,4254
zeroconf/_utils/__init__.py,sha256=Grw4SdBaV4HOx-MzaUL--AYLmxhYxNpOhNLQp7UWyB0,992
zeroconf/_utils/__init__.py,sha256=Grw4SdBaV4HOx-MzaUL--AYLmxhYxNpOhNLQp7UWyB0,992
zeroconf/_utils/name.py,sha256=3SHoeeGOysa_sXvJ62DTaAJQLClr5-5bsIHFBc8rnwg,6979
zeroconf/_utils/name.py,sha256=3SHoeeGOysa_sXvJ62DTaAJQLClr5-5bsIHFBc8rnwg,6979
zeroconf/_utils/net.py,sha256=WX2TkKOJfv2EAT7_hp3FD6gS1nlgleESfE8IrvC9Ajo,15670
zeroconf/_utils/net.py,sha256=WX2TkKOJfv2EAT7_hp3FD6gS1nlgleESfE8IrvC9Ajo,15670
zeroconf/_utils/time.py,sha256=4_gzYha7kLnDKlSEye6jOcSAjJo5cTTxwQD9qFiTaXA,1346
zeroconf/_utils/time.py,sha256=4_gzYha7kLnDKlSEye6jOcSAjJo5cTTxwQD9qFiTaXA,1346
```

Notice that most files appear twice. This only happens in the Windows wheels, all others seem to be fine.

This does not yet cause any problems (that I know of), but it is against the wheel spec. I would like to implement more stringent `RECORD` parsing in pip, though, so I would appreciate if you could fix this.

**Comment by bdraco** [comment]
These are generated via [ciwheelbuild](https://cibuildwheel.readthedocs.io/) so I'm not sure if the problem is here or in ciwheelbuild

**Comment by bdraco** [comment]
Nice work!

**Comment by SpecLad** [comment]
The Poetry bug has been fixed in poetry 1.4.1 / poetry-core 1.5.2.
